pkg/tpl/funcs.go

```diff
@@ -47,6 +48,7 @@ func GetSchemaLink(ref string) string {
 }
 
 func ToHTMLNewLines(str string) string {
+	str = strings.ReplaceAll(str, "\\\n", "<br>")
```

Comment:
I'll recommend to use a replacer like explain here: https://golangdocs.com/replace-characters-in-a-string-in-golang
That will make the code cleaner